src/entities/DFGVariable.jl

```diff
@@ -106,15 +98,15 @@ mutable struct DFGVariable <: AbstractDFGVariable
     label::Symbol
     timestamp::DateTime
     tags::Vector{Symbol}
-    estimateDict::Dict{Symbol, Dict{Symbol, <: AbstractVariableEstimate}}
+    estimateDict::Dict{Symbol, PointParametricEstimates}
```

Comment:
`estimateDict::Dict{Symbol, <: AbstractPointParametric}`

or whatever the chosen name for `AbstractPointParametric` is in the end

Reply:
from #148 

`<: AbstractVariableEstimate`